### PR TITLE
grep: added more freatures and examples

### PR DESCRIPTION
**Pull request by Cvetomird91**
Added the options for recursive search and those for grep'ing a number of lines before or after a match.

### CODE REVIEW

**Comment by ostera** [comment]
can you fix the typo in your commit message?

**Comment by igorshubovych** [comment]
Too much examples.

**Comment by ostera** [comment]
@igorshubovych is right. Didn't see how many examples we had already. In that case I'd say having just -C is more relevant than -A or -B, since it's the union of both.

**Comment by Cvetomird91** [comment]
The -A and -B options have been removed.

File: pages/common/grep.md [hunk 1]
```diff
@@ -11,11 +11,15 @@
 
 `grep -r {{something}} .`
 
+- perform a case insensitive search 
+
+`grep -r {{match}}
```
Comment (igorshubovych) [inline pages/common/grep.md]: According to man, `-r` stands for recursive, not case insensitive search. See previous code example.

Comment (waldyrious) [inline pages/common/grep.md]: Seems to have been a simple overlook.

**Comment by igorshubovych** [comment]
Also run `make lint`. You have a trailing space in grpe page.

**Comment by G570972680893** [comment]
😎

**Comment by Cvetomird91** [comment]
I'm am going to close this pull request. Thank you for your work and moderation.

**Pull request closed without merging**